python/requirements.txt

```diff
@@ -1,4 +1,7 @@
-jinja2 == 3.1.2
+jinja2 == 3.1.3
 numpy == 1.24.4
 pandas == 2.0.3
-pytest == 7.4.2
+pytest == 8.1.1
+coverage == 7.4.4
+pytest-cov == 5.0.0
```

Comment:
Bumped pytest and added coverage / pytest-cov.

I bumped jinja2 too.

pandas and numpy are frozen because we use python 3.8